app/src/main/java/com/scouts/kitchenplaner/ui/navigation/NavHostProjects.kt

```diff
@@ -33,24 +34,19 @@ private const val RECIPE_TO_COOK = "recipeToCook"
 fun NavHostProjects(
     modifier: Modifier = Modifier,
     projectNavController: NavHostController,
-    id: Long = 0L,
+    project: Project,
```

Comment:
Yes, however we kind of have a weird mishmash of Navigation and Composables here. The outer composable (ProjectLayout) already needs the project (for the title and image, but also for the settings sidebar) and it doesn't really make sense to have two flows of the same project coming out of the database (as this would lead to duplicate updates of the composable hierarchy). Then we navigate further into one of the screens. To avoid above mentioned issues, we pass the project down through the navigation here. This is maybe something that can change once we change navigation for the OnRecipeCreated callback, but for now this should (I think) stay this way